model-optimizer/mo/utils/custom_replacement_config.py

```diff
@@ -6,10 +6,12 @@
 import os
 from re import compile, match
 
+import fastjsonschema as json_validate
```

Comment:
please sort imports. Use in PyCharm: Ctrl+A then Ctrl+Alt+O for automatic sorting